dali/pipeline/executor/lowered_graph.cc

```diff
@@ -97,8 +97,18 @@ void OpGraph::Lower(const graph::OpGraph &definition) {
   if (!op_nodes_.empty() || !tensor_nodes_.empty())
     throw std::logic_error("The target graph must be empty");
   for (auto &node : definition.OpNodes()) {
-    auto &lowered_op = AddOp(node.spec, node.instance_name);
-    lowered_op.definition = &node;
+    try {
```

Comment:
So the new executor doesn't use this `OpGraph` representation, right?

Reply:
No, it's the old one - that's why it's now dubbed "lowered" - because it's the new "definition" graph lowered to the old "execution" graph.